### PR TITLE
Get controller.go test coverage to 100%

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -93,6 +93,8 @@ type controller struct {
 	// the queue for processing
 	queue workqueue.RateLimitingInterface
 
+	jitterPeriod time.Duration
```
Comment (droot) [inline pkg/controller/controller.go]: May be a comment explaining what is this really for ?

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -145,21 +149,33 @@ func (c *controller) Start(stop <-chan struct{}) error {
 	for _, informer := range allInformers {
 		syncedFuncs = append(syncedFuncs, informer.HasSynced)
 	}
-	if ok := cache.WaitForCacheSync(stop, syncedFuncs...); !ok {
+
+	// Allow waitForCache to be mocked out for tests
```
Comment (droot) [inline pkg/controller/controller.go]: May be move this comment where this is declared ?

File: pkg/controller/controller.go [hunk 3]
```diff
@@ -145,21 +149,33 @@ func (c *controller) Start(stop <-chan struct{}) error {
...
+	}
+
 	// Launch two workers to process resources
 	log.Info("Starting workers", "controller", c.name, "WorkerCount", c.maxConcurrentReconciles)
 	for i := 0; i < c.maxConcurrentReconciles; i++ {
-		// Continually process work items
+		// Process work items
 		go wait.Until(func() {
-			// TODO(pwittrock): Should we really use wait.Until to continuously restart this if it exits?
+			fmt.Printf("Running\n")
```
Comment (droot) [inline pkg/controller/controller.go]: fmt --> log ?

File: pkg/controller/controller.go [hunk 4]
```diff
@@ -145,21 +149,33 @@ func (c *controller) Start(stop <-chan struct{}) error {
...
 			for c.processNextWorkItem() {
 			}
-		}, time.Second, stop)
+			fmt.Printf("Stopping\n")
```
Comment (droot) [inline pkg/controller/controller.go]: fmt --> log ?

File: pkg/controller/controller_manger_test.go [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+	AfterEach(func() {
+	})
+
+	Describe("", func() {
+
+		It("should do something", func() {
+
+		})
+	})
+})
```
Comment (droot) [inline pkg/controller/controller_manger_test.go]: nit: filename has "manger" --> manager :)

File: pkg/controller/controller.go (hunk 3 above)
Comment (pwittrock) [reply pkg/controller/controller.go]: These were debug statements :P

File: pkg/controller/controller_manger_test.go (hunk 5 above)
Comment (fanzhangio) [reply pkg/controller/controller_manger_test.go]: Should `Describe` be "controller manager", and `//TODO` comments be needed ?

**Pull request merged**